repos/dir.props

```diff
@@ -46,6 +46,11 @@
     <EnvironmentVariables Include="DotNetBuildOffline=true" Condition="'$(OfflineBuild)' == 'true'" />
     <EnvironmentVariables Include="DotNetCoreSdkDir=$(DotNetCliToolDir)" />
     <EnvironmentVariables Include="DotNetBuildToolsDir=$(ProjectDir)Tools/" />
+    <!-- Always build source-build as stabilized.  We don't publish packages so this
+         won't cause conflicts on package feeds, and it will make us match up for releases.  -->
```

Comment:
Do we really want to hard-code these values as true/rc1? I think we want to have them be parameterized somehow. Also I think I would prefer they get passed as part of the command line instead of the env variables but I'd say the same for most of these properties so I can live with it for now.

Reply:
Added parameters for these.  I looked at doing this as part of the build.sh script but we'd need to have some way to pass settings from the non-tarball build to the tarball build, and later when we fix incremental build we'd have to do something similar to keep these the same between rebuilds.